2020/day4/solution.py

```diff
@@ -0,0 +1,57 @@
+import re
+kvp_matcher = re.compile(r"(\S+):(\S+)")
+
+class Passport:
+    RequiredPropertyValidatorMap = {
+        'byr' : lambda x: int(x) >= 1920 and int(x) <= 2002,
```

Comment:
You can re-write all of these using chained comparisons - https://docs.python.org/3/reference/expressions.html#comparisons

e.g. `1920 <= int(x) <= 2002`